test/api/test_workflows.py

```diff
@@ -1234,6 +1236,59 @@ def test_workflow_flatten_with_mapped_over_execution(self):
             assert nested_collection['elements'][0]['object']['populated']
             assert nested_collection['elements'][0]['object']['element_count'] == 2
 
+    @skip_without_tool("cat")
+    def test_workflow_invocation_report_1(self):
+        test_data = """
+input_1:
+  value: 1.bed
```

Comment:
For these tests, but also for workflow tests and user-exploratory-research-things, it would be super useful if we can have default inputs defined in a workflow.

Reply:
We will need this for tools and workflows if we wish to implement CWL - xref https://github.com/galaxyproject/galaxy/pull/1683. Probably not the place for this discussion though.